### ISSUE TITLE
MGW2 (DMWG03LM), ValueError: invalid literal for int() with base 10: b

### ISSUE THREAD

**Issue by garry0garry** [AlexxIT/XiaomiGateway3#891]
DMWG03LM

```
Logger: custom_components.xiaomi_gateway3.core.gateway
Source: custom_components/xiaomi_gateway3/core/gateway/gate_mgw2.py:81
Integration: Xiaomi Gateway 3 (documentation, issues)
First occurred: 22:04:17 (2 occurrences)
Last logged: 22:09:17

192.168.50.76 | Can't update gateway stats
Traceback (most recent call last):
  File "/config/custom_components/xiaomi_gateway3/core/gateway/gate_mgw2.py", line 81, in mgw2_update_stats
    "rssi": int(rssi) + 100
ValueError: invalid literal for int() with base 10: b''
```

**Comment by Joee-D** [comment]
fixed #885 #889

**Comment by Joee-D** [comment]
MR #889  has been conflicted with MR #885,  it should use only one of these two MRs.

2022-12-23 19:12:45.616 WARNING (MainThread) [custom_components.xiaomi_gateway3.core.gateway] 192.168.1.10 | Can't update gateway stats
Traceback (most recent call last):
File "/config/custom_components/xiaomi_gateway3/core/gateway/gate_mgw2.py", line 82, in mgw2_update_stats
"rssi": int(rssi) + 100 if len(rssi) >=1 else 0
TypeError: object of type 'int' has no len()
